src/Common/tests/Common.Tests.builds

```diff
@@ -3,9 +3,11 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
     <Project Include="Common.Tests.csproj">
+      <TestTFMs>netcoreapp1.0;net46;net461;net462;net463</TestTFMs>
       <OSGroup>Windows_NT</OSGroup>
     </Project>
     <Project Include="Common.Tests.csproj">
+      <TestTFMs>netcoreapp1.0</TestTFMs>
```

Comment:
><TestTFMs>netcoreapp1.0</TestTFMs> [](start = 6, length = 34)

this is optional if we have just one TestTFM value and it is netcoreapp1.0. I am ok to leave it as it is but I wanted to mention you'll get the same effect even if you removed this line